modular_chomp/code/modules/mob/living/carbon/human/species/station/station_ch.dm

```diff
@@ -47,4 +47,189 @@
 
 	breath_heat_level_1 = 430	//Default 380 - Higher is better
 	breath_heat_level_2 = 510	//Default 450
-	breath_heat_level_3 = 1200	//Default 1250
\ No newline at end of file
+	breath_heat_level_3 = 1200	//Default 1250
+
+
+/datum/species/xenomorph_hybrid
+	name = SPECIES_XENOMORPH_HYBRID
+	name_plural = "Xenomorph Hybrids"
+	icobase = 'modular_chomp/icons/mob/human_races/r_xenomorph_hybrid.dmi'
+	deform = 'modular_chomp/icons/mob/human_races/r_xenomorph_hybrid.dmi'
+	tail = "tail"
+	icobase_tail = 1
+	unarmed_types = list(/datum/unarmed_attack/stomp, /datum/unarmed_attack/kick, /datum/unarmed_attack/claws, /datum/unarmed_attack/bite/sharp) // Innate claws and bite.
+	darksight = 8 // Same as Tajarans.
+	ambiguous_genders = TRUE
+	gluttonous = 1
+	slowdown = -0.2 // Slightly faster than average.
+	total_health = 150 // Takes a lot of effort to take a Xenomorph down.
+	brute_mod = 0.90 // Physical damage doesn't phase them as much.
+	burn_mod = 1.40 // They do NOT like fire.
+	toxins_mod = 0.90 // Slightly resistant to toxins.
+	radiation_mod = 0.7 // Quite resistant to radiation exposure.
+	flash_mod = 1.1 // Flashes are slightly more effective.
+	metabolic_rate = 1.2 // Very physically active species, thus requiring more nutritional intake.
+	item_slowdown_mod = 0.90 // They carry heavy things slightly better.
+	mob_size = MOB_LARGE

```

Comment:
Keep the default mob_size. Changing this on carbons is fucky and likely to have many unintended consequences.

Reply:
Changed to MOB_MEDIUM